src/button-groups/button-group.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, Input, Output, EventEmitter, ChangeDetectionStrategy, ElementRef, Renderer } from 'angular2/core';
+import { BehaviorSubject } from 'rxjs/subject/BehaviorSubject';
+
+@Component({
```

Comment:
You say so, in order to be able to define a more generic container (div, span, form or whatever)?